scripts/driver.requirements.txt

```diff
@@ -0,0 +1,9 @@
+# Python package requirements for driver implementers.
+
+# Use the version of Jinja that's in Ubuntu 20.04.
+# See https://github.com/ARMmbed/mbedtls/pull/5067#discussion_r738794607 .
+# Note that Jinja2 3.0 drops support for Python 3.5, so we need 2.x.
+Jinja2 >= 2.10.1
```

Comment:
I don't think so: pip is aware of which python versions a package supports, so if left undirected, it will install the latest version (which is now 3.x) on Python ≥3.6, but the latest version that's compatible with Python 3.5 (which is 2.11.x) on Python 3.5. My comment was unclear, I meant that we must not require 3.0 or above as a minimum version since we need to allow a version that is compatible with Python 3.5.

Our code should be compatible with either Jinja 2.{10..11} or Jinja 3.x. Eventually we'll want to test multiple versions, but right now we're focusing on minimum versions.

Reply:
OK, thanks for clarifying the comment.